db/sqldb/desired_lrp_db.go

```diff
@@ -205,6 +205,60 @@ func (db *SQLDB) DesiredLRPSchedulingInfos(ctx context.Context, logger lager.Log
 	return results, err
 }
 
+func (db *SQLDB) DesiredLRPRoutingInfos(ctx context.Context, logger lager.Logger, filter models.DesiredLRPFilter) ([]*models.DesiredLRP, error) {
+	logger = logger.Session("db-desired-lrps-routing-infos", lager.Data{"filter": filter})
+	logger.Debug("starting")
+	defer logger.Debug("complete")
+
+	var wheres []string
+	var values []interface{}
+
+	if filter.Domain != "" {
```

Comment:
Perhaps this 'if' statement is not needed as we do not pass 'Domain' as filtering element in route-emitter